### DISCUSSION TITLE
Verified teacher: cannot access copilot

### DISCUSSION THREAD

**Discussion by paulvickers** [community/community#109451] [GitHub Education] [answered]
### Select Topic Area

Question

### Body

Hi

I've verified my teacher status (twice now, since November) but I still cannot access Copilot (it offers me only the paid plan or a 30-day free trial). I haven't received any coupons.

Can you help?

Thanks

paul

**Comment by avsmith** [comment]
Same here. Did you resolve? (Forgive me for hitting multiple threads on this topic).

**Comment by paulvickers** [comment, marked as answer]
It mysteriously resolved a couple of days after posting on the forum…

From: Albert Vernon Smith ***@***.***>
Date: Monday, 4 March 2024 at 16:50
To: community/community ***@***.***>
Cc: Paul Vickers ***@***.***>, Author ***@***.***>
Subject: Re: [community/community] Verified teacher: cannot access copilot (Discussion #109451)
CAUTION: This email originated from outside of the University. Do not click links or open attachments unless you recognise the sender and know the content is safe.


Same here. Did you resolve? (Forgive me for hitting multiple threads on this topic).

—
Reply to this email directly, view it on GitHub<https://github.com/orgs/community/discussions/109451#discussioncomment-8669446>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ABE5ZMT7R4WPRX7ZOWUN6Q3YWSQ3NAVCNFSM6AAAAABDS5GIE6VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DMNRZGQ2DM>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

This message is intended solely for the addressee and may contain confidential and/or legally privileged information. Any use, disclosure or reproduction without the sender’s explicit consent is unauthorised and may be unlawful. If you have received this message in error, please notify Northumbria University immediately and permanently delete it. Any views or opinions expressed in this message are solely those of the author and do not necessarily represent those of the University. Northumbria University email is provided by Microsoft Office365 and is hosted within the EEA, although some information may be replicated globally for backup purposes. The University cannot guarantee that this message or any attachment is virus free or has not been intercepted and/or amended.

**Reply by avsmith** [reply]
Thanks. Unclear if my reply to your issue will fix mine. I will try my own miraculous query.

**Comment by kowarschick** [comment]
Same here.
